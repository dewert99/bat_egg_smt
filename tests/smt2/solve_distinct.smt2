(declare-sort T)
(declare-const a T)
(declare-const b T)
(declare-const c T)
(declare-const d T)
(assert (distinct a b c))
(assert (not (distinct a b d)))
(check-sat)
(get-model)
(assert (not (distinct c d)))
(check-sat)