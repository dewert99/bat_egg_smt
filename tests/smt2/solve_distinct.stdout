sat
(
 (define-fun a () T T!val!2)
 (define-fun b () T T!val!3)
 (define-fun c () T T!val!4)
 (define-fun d () T T!val!3)
)
unsat
